### ISSUE TITLE
Escaping Special Characters in URLs in Angle Brackets

### ISSUE THREAD

**Issue by oztalha** [retorquere/zotero-citations#21]
GFM appears to be handling the escapes in URLs in angle brackets, but not the HTML/PDF plugins on Atom (at least the most popular ones I have tested, i.e. I tested four preview/export plugins and every plugin kept the backslashes in the URL inside angle brackets instead of evaluating them as escape characters):

Without angle brackets:
https://www.whitehouse.gov/the\-press\-office/2016/01/16/president\-obama\-signs\-michigan\-emergency\-declaration

With angle brackets surrounding the same URL above:
<https://www.whitehouse.gov/the\-press\-office/2016/01/16/president\-obama\-signs\-michigan\-emergency\-declaration>

When I hover over the reference, the URLs in the tip are correct, but in the bibliography, they have extra backslash/escape characters rendered by markdown-preview, MPP, Katex and markdown-pdf.

Please see the URL in the bottom of the screenshot and the one in the hint:
![bibliography vs hover](https://cloud.githubusercontent.com/assets/5094153/15829822/6d9a9188-2be3-11e6-89f3-b05b38a7bb26.png)

**Comment by retorquere** [comment]
I have my defense tomorrow; I'm going to look at this in the weekend.

**Comment by retorquere** [comment]
This is fixed server-side in BBT; let's keep this issue open until that release is out and you can verify the change

**Comment by retorquere** [comment]
Can you try https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-1.6.65-circle-2554.xpi?

**Comment by oztalha** [comment]
Thank you for the follow up.

I am on a different computer now. Don't know why but scan command does not
work on my Mac for some reason (I just tried it after updating to 0.16v).
Currently, nothing happens when I scan.

Besides, I started using zotero-citations as a picker only. Then, within
Atom, using Markdown-Preview-Plus (MPP), I can preview it along with the
bibliography (MPP uses pandoc as a parser, and in the settings
accepts csl and bib files as parameters). Per your understandable
uneasiness on file modification, MPP indeed does not modify the markdown
file. It just shows the bibliography in the preview mode.

-Talha
